package.json

```diff
@@ -25,6 +25,7 @@
     "packages/*",
     "packages/*/*",
     "examples/*",
+    "examples/getstarted/src/plugins/*",
```

Comment:
should we move it into a folder outside the getstarted to make the workspace usecase more obvious ? 

`examples/plugins/workspace-plugin` ?